docs/tutorials/plotting/pairplot_with_gmm.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pairplot with GMM: Visualizing High Dimensional Data\n",
+    "\n",
+    "This example provides how to visualize high dimensional data using the pairplot_with_gmm. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import graspologic\n",
+    "\n",
+    "import numpy as np\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulate a binary graph using stochastic block model\n",
+    "The 3-block model is defined as below:\n",
+    "\n",
+    "\\begin{align*}\n",
+    "n &= [50, 50, 50]\\\\\n",
+    "P &= \n",
+    "\\begin{bmatrix}0.5 & 0.1 & 0.05 \\\\\n",
+    "0.1 & 0.4 & 0.15 \\\\\n",
+    "0.05 & 0.15 & 0.3\n",
+    "\\end{bmatrix}\n",
+    "\\end{align*}\n",
```

Comment:
do we actually need to have this matrix in latex when it's shown in the code?

Reply:
@loftusa I guess not.. just was there for readability... should i take it out? too distraacting? @loftusa

Reply:
I think so? just to stay within the general framework of "be as concise as possible"

Reply:
fix https://github.com/microsoft/graspologic/pull/541/commits/ed819eead52eb3bf208308105c2ad5924a31e2f0